app/controllers/shopify_app/app_proxy_controller.rb

```diff
@@ -0,0 +1,9 @@
+module ShopifyApp
+  class AppProxyController < ApplicationController
+    include ShopifyApp::AppProxyVerification
+
+    skip_before_action :verify_authenticity_token
+
+    before_action :verify_proxy_request
+  end
+end
```

Comment:
The idea was that people would subclass the controller, not edit it directly. It makes sense to me to have all of your controllers that are behind an app proxy to inherit from `ShopifyApp::AppProxyController`. However, once I add the `included do` hook that we talked about, it will be simple to `include AppProxyVerification` from those controllers.